account_payment_order/models/account_payment_mode.py

```diff
@@ -56,6 +56,14 @@ class AccountPaymentMode(models.Model):
         ],
         string="Default Payment Execution Date",
     )
+    default_transfer_move_date_prefered = fields.Selection(
```

Comment:
`default_` prefix is for other kind of fields.

```suggestion
    transfer_date_option = fields.Selection(
```